package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "higlass",
-  "version": "1.5.1",
+  "version": "1.5.2",
```

Comment:
Shouldn't this be v1.5.3?

Reply:
That will sort itself out when merged with master :-)